contracts/token/ERC721/ERC721Token.sol

```diff
@@ -11,15 +12,32 @@ import "zos-lib/contracts/migrations/Migratable.sol";
  * Moreover, it includes approve all functionality using operator terminology
  * @dev see https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
  */
-contract ERC721Token is Migratable, ERC721, ERC721BasicToken {
+contract ERC721Token is Migratable, SupportsInterfaceWithLookup, ERC721BasicToken, ERC721 {
```

Comment:
`ERC721BasicToken`'s storage moved after `SupportsInterfaceWithLookup`'s.

That can be fixed by declaring
```
contract ERC721Token is Migratable, ERC721BasicToken, SupportsInterfaceWithLookup, ERC721 {
```

but we still have the same problem I mentioned for `ERC721BasicToken.